From 1866952edae5d4adabfc2fc32c2bde930af8cbe3 Mon Sep 17 00:00:00 2001
From: Sebastian Lackner <sebastian@fds-team.de>
Date: Tue, 9 Jun 2015 06:14:49 +0200
Subject: include: Fix definition of FILE_DISPOSITION_INFO.

A member called DeleteFile cannot be accessed because its defined as
DeleteFile_must_be_suffixed_with_W_or_A_in_this_context.
---
 include/winbase.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/winbase.h b/include/winbase.h
index 4795d87..44ecc92 100644
--- a/include/winbase.h
+++ b/include/winbase.h
@@ -860,7 +860,7 @@ typedef struct _FILE_ALLOCATION_INFO {
 } FILE_ALLOCATION_INFO, *PFILE_ALLOCATION_INFO;
 
 typedef struct _FILE_DISPOSITION_INFO {
-    BOOLEAN DeleteFile;
+    BOOLEAN DoDeleteFile;
 } FILE_DISPOSITION_INFO, *PFILE_DISPOSITION_INFO;
 
 typedef struct _FILE_END_OF_FILE_INFO {
-- 
2.4.2

